src/containers/Token/index.tsx

```diff
@@ -84,20 +85,6 @@ const Token = ({ error }) => {
   )
 }
 
-Token.propTypes = {
-  error: PropTypes.number,
-  match: PropTypes.shape({
-    params: PropTypes.shape({
-      id: PropTypes.string,
-      currency: PropTypes.string,
-    }),
-  }).isRequired,
-}
-
-Token.defaultProps = {
-  error: null,
-}
-
-export default connect((state) => ({
+export default connect((state: any) => ({
```

Comment:
> This also required updating some types that were wrong. All of the any types were done because it was already set to a prop type of essentially any.

Also this is for redux which will be removed shortly.